src/go/rdctl/pkg/shutdown/shutdown.go

```diff
@@ -57,7 +57,7 @@ func FinishShutdown(waitForShutdown bool, initiatingCommand InitiatingCommand) e
 	}
 	var err error
 	if err = directories.SetupLimaHome(); err != nil {
-		return err
+		logrus.Errorf("Ignoring error trying to get lima directory: %s", err)
```

Comment:
This is wrong: when you cannot setup `LIMA_HOME`, then you should not setup `limaCtlPath`, and make sure you don't call it. Because now you will run `limactl` against `~/.lima`, and if the user has an instance with the same name in there, you will kill it.

When `LIMA_HOME` cannot be set, then you need to go straight to `pkill` and not try to use `limactl` at all.

Reply:
Good point. I assume the same holds for the `qemu` process, because it's tied to lima.